packages/dso-toolkit/src/components/description/description.content.ts

```diff
@@ -9,7 +9,7 @@ export const termContent: Description = {
   open: false,
 };
 
-export function descriptionExample(openTerm: boolean) {
+export function descriptionExample(openTerm: boolean): (string | Description)[] {
```

Comment:
De return type van deze functie heb ik expliciet gemaakt. Dit is demo content. De array van items worden achter elkaar geplakt om een lopende zitte maken. Als het een string is, wordt het direct getoond. Als het een `Description` wordt het aan `descriptionTemplate()` gegeven en komt er een Description.